vernac/obligations.ml

```diff
@@ -295,8 +295,8 @@ open Evd
 
 let unfold_entry cst = Hints.HintsUnfoldEntry [EvalConstRef cst]
 
-let add_hint local prg cst =
-  Hints.add_hints ~local [Id.to_string prg.prg_name] (unfold_entry cst)
+let add_hint prg cst =
```

Comment:
What about renaming this one `add_local_hint`?

Reply:
Done.